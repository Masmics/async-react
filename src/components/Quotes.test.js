import React from 'react';
import { shallow } from 'enzyme';
import Quotes from './Quotes';

describe('quotes component', () => {
  it('renders a list of quotes', () => {
    const quotes = [
      {
        character: 'Fry',
        quote: 'What??',
        image: '/image'
      },
      {
        character: 'Fry',
        quote: 'What??',
        image: '/image'
      }
    ];

    const wrapper = shallow(<Quotes quotes={quotes} />);
    expect(wrapper).toMatchSnapshot();
  });
});
