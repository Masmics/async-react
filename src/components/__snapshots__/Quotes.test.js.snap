// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`quotes component renders a list of quotes 1`] = `<ul />`;
